variation-commons-core/src/main/java/uk/ac/ebi/eva/commons/core/models/AbstractVariant.java

```diff
@@ -29,6 +29,14 @@
 
     public static final int SV_THRESHOLD = 50;
 
+    public static final String VARIANT_START_COORDINATE_CANNOT_BE_NEGATIVE =
```

Comment:
If they are only going to be used internally or from tests, these constants can be packaged-protected.